src/main/kotlin/baseball/Application.kt

```diff
@@ -1,5 +1,98 @@
 package baseball
 
-fun main() {
-    TODO("프로그램 구현")
-}
+import camp.nextstep.edu.missionutils.Console
+import camp.nextstep.edu.missionutils.Randoms
+import java.lang.IllegalArgumentException
+
+    private fun inputException(userNumber: MutableList<Char>, inputNumber: String) {
+        if (inputNumber.length != 3) {
+            throw IllegalArgumentException("입력된 숫자가 3자리가 아닙니다.")
+        }
+        if (!inputNumber.matches(Regex("\\d+"))||inputNumber.matches(Regex("^0$"))){
+            throw IllegalArgumentException("숫자가 아닌 문자 또는 0이 포함되어있습니다.")
+        }
+        val uniqueNumbers = inputNumber.toSet().size
+        if (uniqueNumbers != inputNumber.length) {
```

Comment:
uniqueNumbers 가 불필요 해보입니다.
uniqueNumbers 라는 변수를 만들 필요 없이 1줄로 만드시면 보다 깔끔하게 할 수 있을 것 같습니다.
```suggestion
        if ( inputNumber.toSet().size!= inputNumber.length) {
```